.devcontainer/devcontainer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Benchpress Dev Environment",
-  "image": "mcr.microsoft.com/devcontainers/base:bullseye",
+  "image": "mcr.microsoft.com/dotnet/aspnet:6.0-bullseye-slim",
```

Comment:
instead of changing the base image, can we add dotnet in the "features" list? what do you think?

Reply:
Unless I'm misunderstanding your suggestion, the problem is the existing image is ,NET 7. The updated image uses .NET 6, which is necessary to build the .NET solution.

Reply:
Unfortunately, GH cut off my complete commit message, the first line of which is:

Updated the devcontainer to use an image that supports .NET 6.0 so that Generators builds.

Reply:
I think it'd be better if we don't have dotnet as the base image and one of the features that'll be installed.

and it seems "dotnet" feature accepts "version" parameter; https://github.com/devcontainers/features/tree/main/src/dotnet

if we need .net 6 to be installed on the container, we can add it as;

```json
"ghcr.io/devcontainers/features/dotnet:1": {
  "version": "6"
}
```
Here is what I found about the version parameter; https://github.com/devcontainers/features/blob/main/src/dotnet/devcontainer-feature.json#L13

Reply:
Ah, gotcha. I'm pretty new to devcontainers. The research I had done suggested that devcontainers/base:bullseye was a .NET 7 image. I didn't realize that the .NET version could be manipulated simply with the listed featurs.
 
 The requested changes have been made.